server/endpoints.ts

```diff
@@ -5,6 +5,7 @@ import { profileRouter } from "./src/profile";
 import { reviewRouter } from "./src/review";
 import { courseRouter } from "./src/course";
 import { adminRouter } from "./src/admin";
+import aiRouter from "./src/ai/ai.router";
 
 export const configure = (app: Express) => {
   app.use(
```

Comment:
one thing we can do is 

`app.use('/ai', aiRouter)`

so we prefix `/ai` in front of all our ai endpoints